ci/test_python.sh

```diff
@@ -50,6 +41,19 @@ set +e
 # bulk sampler IO tests (hangs in CI)
 
 if [[ "${RUNNER_ARCH}" != "ARM64" ]]; then
+  rapids-logger "Generate Python testing dependencies"
+  rapids-dependency-file-generator \
+    --output conda \
+    --file-key test_cugraph_dgl \
+    --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}"  \
+    --prepend-channel "${CPP_CHANNEL}" \
+    --prepend-channel "${PYTHON_CHANNEL}" \
+    --prepend-channel pytorch \
+    --prepend-channel conda-forge \
+    --prepend-channel "${DGL_CHANNEL}" \
```

Comment:
`DGL_CHANNEL` is computed further below, in that block beginning

```shell
 if [[ "${RAPIDS_CUDA_VERSION%%.*}" == "11" ]]; then
```

That needs to be moved up above this.